src/test/java/br/com/caelum/comidinhas/tipoCozinha/NovoTipoCozinhaValidatorTest.java

```diff
@@ -0,0 +1,39 @@
+package br.com.caelum.comidinhas.tipoCozinha;
+
+import org.junit.jupiter.api.*;
+import org.springframework.validation.Errors;
+
+import static org.mockito.Mockito.*;
+
+class NovoTipoCozinhaValidatorTest {
+
+    private TipoCozinhaRepository tipoCozinhaRepository;
+    private Errors errors;
+    private TipoCozinhaInputNovo tipoCozinhaInputNovo;
+    private NovoTipoCozinhaValidator novoTipoCozinhaValidator;
+
+    @BeforeEach
+    public void setup(){
```

Comment:
O método `setup` não precisa ser público.

```suggestion
    void setup(){
```